docs-ref-conceptual/get-started-tutorial-1-prepare-environment.md

```diff
@@ -0,0 +1,183 @@
+manager: jasongroce
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 11/15/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli
+keywords: azure, az account, az group, az config, az init, configuration
+---
+# Prepare your environment for the Azure CLI
+
+In this tutorial step, you onboard with the Azure CLI! This includes choosing the environment that works best for you, and setting your default values. You will also learn how to find and change your active Azure subscription, and use random IDs to always guarantee unique resource names.
+
+## Install the Azure CLI
+
+* If you don't have an [Azure subscription](/azure/guides/developer/azure-developer-guide#understanding-accounts-subscriptions-and-billing), create an [Azure free account](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio) before you begin.
+
+* Use the Bash environment in [Azure Cloud Shell](/azure/cloud-shell/overview) by selecting the **Open Cloud Shell** button in the top right corner of each Azure CLI code block.
+
+* If you prefer to run the Azure CLI reference commands locally, [install](/cli/azure/install-azure-cli) the Azure CLI.
+
+* To complete the steps in this tutorial, you will need `contributor` permissions in your subscription.
+
+[!INCLUDE [include](./includes/current-version.md)]
+
+## Sign in to Azure using the Azure CLI
+
+There are several [authentication options](./authenticate-azure-cli.md) when working with the Azure CLI. The Azure CLI's default authentication method for logins uses a web browser and access token to sign in.
+
+[!INCLUDE [interactive_login](includes/interactive-login.md)]
+
+## Find and change your active subscription
+
+After logging into the Azure CLI, always check your default subscription. If you aren't working under the subscription you prefer, change your subscription using [az account set](/cli/azure/account#az-account-set). Here is a code example to use.
+
+```azurecli-interactive
+# see your current default subscription
+az account show
+
+# find the list of subscriptions available to you
+az account list --output table
+
+# set your default subscription
+az account set --subscription <mySubscriptionName>
+
+# you can also set your subscription using your subscription ID
+az account set --subscription <00000000-0000-0000-0000-000000000000>
+```
+
+## Learn about resource groups
+
+A resource group is a container for Azure resources. To create a resource group, you need role permissions of `contributor` or higher in your subscription.
+
+### Create a resource group
+
+1. A resource group name must be globally unique. First check if the name you want is available using the [az group exists](/cli/azure/group#az-group-exists) command.
+
+   ```azurecli-interactive
+   az group exists --name <myUniqueRGname>
```

Comment:
It would be helpful for the user to know if the output returns "true" or "false" and what that means. ie: False = name not taken

Reply:
done.  thanks for this insight.